client.go

```diff
@@ -44,18 +43,30 @@ const (
 )
 
 // AkamaiEnvironment represents Akamai's target environment type.
-//
-// client
 type AkamaiEnvironment string
 
 const (
 	Production AkamaiEnvironment = "production"
 	Staging    AkamaiEnvironment = "staging"
 )
 
+// AkamaiRequestFrom represents Akamai's source for request.
+type AkamaiRequestFrom string
+
+const (
```

Comment:
Let's add constants on top of the file and in one place

Reply:
Already fixed https://github.com/apiheat/go-edgegrid/blob/23abfca561129b7d5ffb15dbd6ca01c388f5effc/client.go